.github/workflows/ci.yml

```diff
@@ -71,5 +71,9 @@ jobs:
       - name: Run tests
         run: bundle exec rake beaker
         env:
+          # Yes, this should be in modulesync, but for now, just work around weird test failures
+          # caused by locale on CentOS 7 with Puppet 7
+          LANG: en_US
+          LC_ALL: en_US.UTF-8
```

Comment:
https://github.com/voxpupuli/modulesync_config/pull/708 has more context. Linking to cross-reference.